### PR TITLE
Allow for injection of pod annotations for helper pods.

### PR DESCRIPTION
**Pull request by cmccabe-ws**
Currently only agent pods allow for annotation injection, I also need to be able to inject annotations to helper pods.

### CODE REVIEW

File: charts/s1-agent/values.yaml [hunk 1]
```diff
@@ -80,6 +80,7 @@ helper:
   fullnameOverride: ""
   nameOverride: ""
   labels: {}
+  podAnnotations: ""
```
Comment (andrewjamesbrown) [inline charts/s1-agent/values.yaml:83]:
```suggestion
  podAnnotations: {}
```

Comment (cmccabe-ws) [reply charts/s1-agent/values.yaml:83]: @andrewjamesbrown I was going to complete it this way, however that isn't how the default is set for `podAnnotations` under `agent`.

**Comment by cmccabe-ws** [comment]
@oded-s1 or @danisevas 

I noticed in previous PRs you have approved/merged - would you be able to take a look at this change?

Thanks
Connor M

**Comment by danisevas** [comment]
@DorWeiss @oded-s1 I think it is better that one of you review it.

**Comment by cmccabe-ws** [comment]
@DorWeiss @oded-s1 just a bump on this PR

**Comment by DorWeiss** [comment]
@cmccabe-ws Please remove any merge commits from your branch and use rebase instead.
We will review it this week.

**Comment by cmccabe-ws** [comment]
@DorWeiss completed

**Review by natishauli — APPROVED** [review]

**Pull request merged**